### DISCUSSION TITLE
Could we get a new tagged release?

### DISCUSSION THREAD

**Discussion by ilikenwf** [netblue30/firejail#6506] [General]
I use the -git version for the newer profiles, but even under Archlinux, the package maintainer is a bit asleep at the wheel and hasn't bumped "pkgrel" since 2023.

https://aur.archlinux.org/packages/firejail-git

**Comment by rusty-snake** [comment]
Personally I expect that @netblue30 tags a new release before the next Debian stable freeze for Trixie which can be expected in spring 2025 I guess.

> pkgrel

=`master` ?

AFAIK there are -got AUR packages that always build from latest commit.

**Comment by ilikenwf** [comment]
pkgrel is a non version specific incremented value that the maintainer bumps once in a while to trigger AUR users to rebuild and reinstall the package. It tracks the package version release for the given version of the package...so if the packager screwed something up or needs to bump the package for rebuild, they increment it...

Ideally it gets reset the next time a new release comes out, too.

I'm trying to avoid requesting ownership of the package in the hope that the maintainer wakes up to the fact I flagged it...a ton of changes and profiles since 2023.

To be fair, I can just build the current version with his package, but not having it bumped means that AUR package manager programs don't pick up on the fact a -git package needs updating.
